src/Dapr.Client/SubscribeConfigurationResponse.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Dapr.Client
+{
+    /// <summary>
+    /// Response for a Subscribe Configuration request.
+    /// </summary>
+    [Obsolete("This response utilizes an alpha API which is subject to change. This attribute will be removed when the API is no longer Alpha.")]
+    public class SubscribeConfigurationResponse
```

Comment:
Why do we need this class **and** `ConfigurationSource`. It looks like this could just be replaced with `ConfigurationSource` without losing anything.

Reply:
At this point it was just to keep the similarity with the `GetConfigurationResponse`. I didn't want to give this the class the implementation of the `DaprSubscribeConfigurationSource` because it exposed some weird internals. Now that it's just taking the server stream it's a bit better. 

The other reasoning being I thought it'd be easier for things like testing but I guess you can just have the client give back a difference source anyway.

Reply:
Ok, I think it's fine I was just curious. If you want to keep them separated then lets keep them separated.